bin/aleph

```diff
@@ -33,6 +33,26 @@ class CommandParser
         @options[:worker_processes]  = w
       end
 
+      opts.on('-H', '--redshift-host REDSHIFT_HOST', 'Redshift Hostname') do |rhost|
```

Comment:
easier? seems more confusing to have to put the all host info in one way and specify the secrets another way.